raiden/storage/versions.py

```diff
@@ -2,18 +2,21 @@
 import re
 from glob import glob
 
+from raiden.utils import typing
+
 from .sqlite import RAIDEN_DB_VERSION
 
 VERSION_RE = re.compile("^v(\d+).*")
 
 
-def older_db_files_exist(database_base_path: str):
+def older_db_files_exist(database_base_path: str) -> typing.Optional[str]:
     """ Check if the directory contains database files that
```

Comment:
Can you also update the docstring? That you now return the first of those databases?